src/Client.php

```diff
@@ -8,6 +8,9 @@
 
 class Client
 {
+    protected const STATUS_SUBSCRIBED = 'subscribed';
```

Comment:
I have to remove the constant visibility here due to `PHP >= 5.4` compatibility.